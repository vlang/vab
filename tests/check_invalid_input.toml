execute = 'vab invalid_input'
[expect]
	exit_code = 1

[compare]
	output.from_line = -1 # negative means from bottom, positive from the top
