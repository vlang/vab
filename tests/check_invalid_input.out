vab: input should be a V file, an APK, AAB or a directory containing V sources